www/modules/centreon-open-tickets/views/rules/list.ihtml

```diff
@@ -41,7 +46,7 @@
                 {if $elem.status == 'Disabled'}
                 <tr class="row_disabled">
                 {else}
-                <tr class="row_enabled">
+                <tr class="{cycle values='list_one,list_two'} row_enabled">
```

Comment:
didn't know this smarty trick
this is pretty cool :)

Reply:
thanks :D